diff -dur -x '*~' slony1-1.1.2.orig/tools/slony1_dump.sh slony1-1.1.2/tools/slony1_dump.sh
--- slony1-1.1.2.orig/tools/slony1_dump.sh	2005-09-28 03:18:56.000000000 +0200
+++ slony1-1.1.2/tools/slony1_dump.sh	2005-11-03 10:00:46.000000000 +0100
@@ -177,7 +177,7 @@
 # ----
 echo "select 'copy $clname.sl_sequence_offline from stdin;';"
 echo "select seq_id::text || '	' || seq_relname  || '	' || seq_nspname from $clname.sl_sequence;"
-echo "select '\\\\.';"
+printf "select '\\\\\\\\.';"
 
 for seq in $sequences ; do
 	eval seqname=\$seqname_$seq
@@ -203,7 +203,7 @@
 
 	echo "copy $tabname to stdout;"
 
-	echo "select '\\\\.';"
+	printf "select '\\\\\\\\.';"
 done
 
 # ----
