diff -dur -x '*~' slony1-1.1.2.orig/tools/slony1_dump.sh slony1-1.1.2/tools/slony1_dump.sh
--- slony1-1.1.2.orig/tools/slony1_dump.sh	2005-11-04 13:05:15.000000000 +0100
+++ slony1-1.1.2/tools/slony1_dump.sh	2005-11-04 13:06:43.000000000 +0100
@@ -130,7 +130,7 @@
 -- ---------------------------------------------------------------------------------------
 -- FUNCTION setsyncTracking_offline (seq_id, seq_origin, ev_seqno, last_value, sync_time)
 -- ---------------------------------------------------------------------------------------
-create or replace function $clname.setsyncTracking_offline(int4, int8, int8) returns int8
+create or replace function $clname.setsyncTracking_offline(int4, int8, int8, timestamp) returns int8
 as '
 declare
 	p_set_id	alias for \$1;
