--- slony1-1.1.5/config/acx_libpq.m4.orig	2005-11-01 21:33:04.000000000 +0100
+++ slony1-1.1.5/config/acx_libpq.m4	2006-01-27 11:07:42.000000000 +0100
@@ -185,43 +185,7 @@
     )
 fi
 
-LDFLAGS="$TEMP_LDFLAGS -L$PG_PKGLIBDIR"
-AC_MSG_CHECKING(for plpgsql.so)
-if test -s $PG_PKGLIBDIR"/plpgsql.so"; then
-    AC_MSG_RESULT(yes)
-    AC_DEFINE(PG_PKGLIBDIR_VERIFIED,1,[PostgreSQL pkglibdir])
-elif test -s $PG_PKGLIBDIR"plpgsql.sl"; then
-    AC_MSG_RESULT(yes)
-    AC_DEFINE(PG_PKGLIBDIR_VERIFIED,1,[PostgreSQL pkglibdir])
-elif test -s $PG_PKGLIBDIR"plpgsql.dll"; then
-    AC_MSG_RESULT(yes)
-    AC_DEFINE(PG_PKGLIBDIR_VERIFIED,1,[PostgreSQL pkglibdir])
-else
-    AC_MSG_RESULT(no)
-    AC_MSG_ERROR($PG_PKGLIBDIR/plpgsql.[so|sl|dll] is not found in the pkglibdir.
-    Please specify the pkglibdir with --with-pgpkglibdir=<dir>
-    )
-fi
-
-AC_MSG_CHECKING(for postgresql.conf.sample)
-PGSHARE_POSSIBLE_LOCATIONS="${with_pgsharedir} /usr/local/pgsql/share /usr/local/share/postgresql /usr/share/postgresql /usr/local/share/pgsql /usr/share/pgsql /opt/local/pgsql/share /opt/pgsql/share ${PG_BINDIR}/../share"
-for i in $PGSHARE_POSSIBLE_LOCATIONS; do
-    if test -s "$i/postgresql.conf.sample" ; then
-	PG_SHAREDIR=$i/
-	break;
-    fi
-done
-
-if test -n "$PG_SHAREDIR" ; then
-    AC_MSG_RESULT(${PG_SHAREDIR}postgresql.conf.sample)
-    AC_DEFINE(PG_SHAREDIR_VERIFIED,1,[PostgreSQL sharedir])
-else
-    AC_MSG_RESULT(not found)
-    AC_MSG_ERROR(
-	postgresql.conf.sample not found! Please specify the sharedir
-	with --with-pgsharedir=<dir>
-	)
-fi
+PG_SHAREDIR="${with_pgsharedir}"
 
 LDFLAGS="$TEMP_LDFLAGS -L$PG_LIBDIR"
 
